APIs/SweccAPI.py

```diff
@@ -11,10 +11,10 @@ def __init__(self):
             "Content-Type": "application/json",
         }
         self.reaction_channel_subscriptions = {
-            os.getenv('NG_CHANNEL_ID'),
+            os.getenv('NEW_GRAD_CHANNEL_ID'),
             os.getenv('INTERNSHIPS_CHANNEL_ID')
         }
-        self.COMPLETED_EMOJI = ":white_check_mark:"
+        self.COMPLETED_EMOJI = "✅"
```

Comment:
Was `:white_check_mark:` the wrong way of doing it?

Reply:
i ...think so

![Screenshot 2024-12-27 at 12 37 02 PM](https://github.com/user-attachments/assets/ac495c84-2e6b-45cc-b2b0-1f236f796ae5)